rllib/algorithms/ppo/torch/ppo_torch_rl_trainer.py

```diff
@@ -0,0 +1,135 @@
+import logging
+from typing import Mapping, Any
+
+from ray.rllib.core.rl_trainer.torch.torch_rl_trainer import TorchRLTrainer
+from ray.rllib.evaluation.postprocessing import Postprocessing
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.torch_utils import (
+    explained_variance,
+)
+from ray.rllib.utils.typing import TensorType
+
+torch, nn = try_import_torch()
+
+logger = logging.getLogger(__name__)
+
+
+class PPOTorchRLTrainer(TorchRLTrainer):
+    def __init__(self, *args, **kwargs):
```

Comment:
Please add a dosctring describing what the `PPO`-bit here adds to the `TorchRLTrainer` base-class.

Also, let's add @override (and @OverrideToImplementCustomLogic and @OverrideToImplementCustomLogic_CallToSuperRecommended to the base class(es)) decorators everywhere in this PR so it's more clear why we are overriding these.